charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -101,12 +101,14 @@ insecureScheme:
       #   cpu: 100m
       #   memory: 64Mi
 
+jwtIssuer: oidc-discovery.example.org
+
 config:
   # -- The log level, valid values are "debug", "info", "warn", and "error"
   logLevel: info
+  # -- Override the domains to use. (jwtIssuer is automatically added to this list).
```

Comment:
This kind of comment also reinforces my previous statement that the change makes the configuration "API" not align with what it says it does.  Consider calling the list "additionalDomains".